docs: Updating the reference docs (new workspace) (#1346)
recognai/rubrix: docs/reference/webapp/dataset_list.md

# Dataset list

![Screenshot of the Workspace page](../../_static/reference/webapp/workspace1.png)

The _Dataset list_ page is mainly a **filtrable, searchable and sortable list** of **datasets**.
It is the **entry point** to the Rubrix web app and is composed of the following three components.

## Search bar

The "_Search datasets_" bar on the top allows you to search for a specific dataset by its name.

## Dataset list

In the center of the page you see the list of datasets available for **your user account**.
The list consists of following columns:

- **Name**: The name of the dataset, can be sorted alphabetically.

> - **Workspace**: A filtrable column in which the datasets of one or different workspaces can be chosen to be seen.

Reviewer: ```suggestion
- **Workspace**: A filtrable column showing the workspace to which the dataset belongs.
```

- **Task**: The [task](../../guides/task_examples.ipynb) of the dataset. This is a filtrable column.
- **Tags**: User defined tags for the dataset.
- **Created at**: When was the dataset first logged by the client.
- **Updated at**: When was the dataset last modified, either via the Rubrix web app or the client.

## Side bar

You can find a user icon and a refresh button on the top right:

> - **User icon**: This icon, which shows the initials of your username, allows you **log out** or **view the reference docs**.

Reviewer: ```suggestion
- **User icon**: This icon shows the initials of your username and allows you to **log out** or **view the reference docs**.
```
